lcls-twincat-pmps/PMPS/MajorComponents/Arbiter/FB_Arbiter.TcPOU

```diff
@@ -40,22 +40,19 @@ VAR
         field: DESC Arbiter ID for elev. req.
      '}
     nArbiterID : UDINT; // Arbiter ID, used for making higher-level BP requests
+    
+    nNextCohort : UDINT := 1; // The cohort ID any new requests will adopt, will become ReqInProgCohort at the start of the next acknowledgement cycle
+    nAckInProgCohort : UDINT := 0; // The cohort ID currently being acknowledged, will become nActiveCohort after acknowledgement from HA
     {attribute 'pytmc' := '
         pv: CohortCounter
         io: i
         field: DESC Intrnl cohort counter
      '}
-    nArbiterCohort : UDINT := 1; // The cohort index keeps track of which beam parameter requests have been included in higher-level arbitrations
-    // Init to 1, provides a guard against default cohort value in BP requests.
-    {attribute 'pytmc' := '
-        pv: ActiveInArbitration
-        io: i
-        field: DESC True if this arbiter's req is included
-     '}
```

Comment:
It looks like we lost one PV, was this important for the UI at all?

Reply:
Completely vestigial.